xbmcaddon.py

```diff
@@ -220,15 +244,46 @@ def __get_strings(self):
 
         return translations
 
+    def __filter_lang_xmltag(self, tag, xml_content):
+        tag_matches = re.findall(r'<' + tag + r'(?:\s*lang=\"([a-zA-Z-]+)\")?\s*>(.*?)</' + tag + '>', xml_content, flags=re.DOTALL)
+        if tag_matches:
+            for match in tag_matches:
+                if match[0].startswith('en'):
+                    return match[1]
+            else:
+                return tag_matches[0][1]
+        return None
+
     def __load_add_on_xml(self):
         add_on_xml = os.path.join(self.__add_on_path, "addon.xml")
 
         with io.open(add_on_xml, encoding='utf-8') as fp:
             xml_content = fp.read()
-            self.__version = re.findall(r'version="(\d+.\d+.\d+[^"]*)', xml_content)[0]
+            self.__version = re.findall(r'<addon.*?version="(\d+(?:\.\d)?(?:\.\d)?[^"]*)', xml_content, flags=re.DOTALL)[0]
```

Comment:
The regex won't work for all version. Better maken it `r'<addon.*?version="([^"]*)'` which will cover it all.